Mathlib/Mem.lean

```diff
@@ -0,0 +1,6 @@
+class Mem (α : outParam $ Type u) (γ : Type v) :=
+(mem : α → γ → Prop)
+
+infix:50 " ∈ " => Mem.mem
+
+notation:50 x "∉" s => ¬ x ∈ s
```

Comment:
```suggestion
notation:50 x " ∉ " s => ¬ x ∈ s
```
The extra spaces here tell the pretty-printer to include spaces when pretty-printing, otherwise it would print `x∉s` instead of `x ∉ s`.

(It's not necessary to add spaces for words, `"in"` will automatically get spaces.)

Reply:
Done